core/src/main/java/tech/tablesaw/api/BooleanColumn.java

```diff
@@ -297,12 +296,12 @@ public BooleanColumn copy() {
 
   @Override
   public void sortAscending() {
-    ByteArrays.mergeSort(data.elements());
+    data.sort(descendingByteComparator.reversed() /* or ByteComparators.NATURAL_COMPARATOR */);
   }
 
   @Override
   public void sortDescending() {
-    ByteArrays.mergeSort(data.elements(), descendingByteComparator);
+    data.sort(descendingByteComparator);
```

Comment:
Perhaps we should remove `descendingByteComparator` and use `ByteComparators.OPPOSITE_COMPARATOR` here for consistency if we change `sortAscending`